server/models/user.js

```diff
@@ -0,0 +1,41 @@
+const { DataTypes } = require("sequelize");
+const bcrypt = require("bcryptjs");
+const db = require("../db");
+
+const User = db.define("User", {
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    get() {
+      // this prevents password from being serialized on the response output
+      // because the password field is now a getter function
+      // Therefore to access the value stored you must call it as:
+      // someUserModel.password()
+      return () => this.getDataValue("password");
+    },
+  },
+});
```

Comment:
According to the signup page, we may need a "company" field here.

Reply:
Oh right, will be pushing for that change now